src/app/app.component.ts

```diff
@@ -1,10 +1,33 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ReactiveFormsModule, FormControl } from '@angular/forms';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+// import {MatButtonModule, MatFormFieldModule, MatInputModule, MatRippleModule} from '@angular/material'
+import {MatFormFieldModule} from '@angular/material/form-field';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
-  title = 'angular-v8';
+export class AppComponent implements OnInit {
+  title = 'Feature Im Working On';
+  constructor(private fb: FormBuilder){}
+  myForm: FormGroup;
+
+  ngOnInit() {
+    this.myForm = this.fb.group({
+      name: ['Benedict', Validators.required],
+      email: ['', [Validators.required, Validators.pattern('[a-z0-9.@]*')]],
```

Comment:
We should use the Validators.email instead of a custom pattern

see: 
https://angular.io/api/forms/Validators